finetune/config.py

```diff
@@ -400,6 +401,7 @@ def get_default_config():
         mlm_baseline=False,  # single projection after last layer of base model
         n_layers_with_aux=0,  # if -1, add to all layers
         mlm_context_shuffle_val=False,
+        seq_labeling_2d_spans=False,
```

Comment:
Can you add a description in the docstring for posterity? Something like "combine 2d sequence labeling spans into contiguous chunks"